### PR TITLE
*: move the code of distinct logic.

### PR DESCRIPTION
**Pull request by hanfei1991**
refactor
@shenli @coocood @zimulala @coocood PTAL

### CODE REVIEW

File: plan/physical_plan_builder.go [hunk 1]
```diff
@@ -32,7 +32,6 @@ const (
 	netWorkFactor   = 1.5
 	memoryFactor    = 5.0
 	selectionFactor = 0.8
-	distinctFactor  = 0.7
```
Comment (shenli) [inline plan/physical_plan_builder.go]: Why remove this?

Comment (hanfei1991) [inline plan/physical_plan_builder.go]: No Longer used

**Comment by shenli** [comment]
LGTM
@coocood @zimulala @tiancaiamao PTAL

**Comment by zimulala** [comment]
LGTM

**Pull request merged**